rdr_service/code_constants.py

```diff
@@ -97,6 +97,9 @@
 CONSENT_COPE_NO_CODE = "COPE_A_13"
 CONSENT_COPE_DEFERRED_CODE = "COPE_A_231"
 
+# COPE Minute Survey Codes
+VACCINE_MINUTE_1_MODULE_CODE = "cope_vaccine1"
```

Comment:
Wondering if we should include COPE_ in the named constants, etc.   (On the off chance there is some future module pertaining to a different vaccine)

Reply:
I think I might have started with that, but took it out since it was a bit long. At the moment there's only the one minute survey, and a constant in the code would be easy enough to update if another minute survey was started for another vaccine. But I can update it if you still think it'd be better to have it contain COPE now rather than later.

Reply:
Added COPE